frontend/.prettierrc

```diff
@@ -2,5 +2,6 @@
   "trailingComma": "es5",
   "semi": false,
   "singleQuote": true,
-  "printWidth": 120
+  "printWidth": 120,
+  "tabWidth": 2
```

Comment:
What does adding `tabWith: 2` do?

Reply:
It enforces that tabs are 2 spaces, rather than the default 4